roles/kubelet/templates/logspout.yml.j2

```diff
@@ -19,7 +19,7 @@ spec:
     - name: SYSLOG_HOSTNAME
       value: '{% raw %}{{ index .Container.Config.Labels "io.kubernetes.pod.namespace" }}{% endraw %}'
     - name: SYSLOG_TAG
-      value: 'klondike-{{ cluster }}-{% raw %}{{ index .Container.Config.Labels "io.kubernetes.pod.namespace" }}-{{ index .Container.Config.Labels "io.kubernetes.pod.name" }}{% endraw %}'
```

Comment:
and I would go one step further and append the container name to the end, but thats a discussion for another review